README.md

```diff
@@ -31,31 +31,31 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
-
+ ` Dom is the document Object `
 2. What is an event?
-
+ ` An event is the ability of a DOM object to interact with the user `
 3. What is an event listener?
-
+ ` An event listener is an event trigerred by interaction `
 4. Why would we convert a NodeList into an Array?
-
+ ` You might want to iterate `
```

Comment:
To gain access to more methods, which are available to arrays. ( .forEach() is available to a Nodelist and could still "iterate" )